src/utils/TransactionUtils.ts

```diff
@@ -20,7 +21,11 @@ export async function runTransaction(
   nonce: number | null = null
 ): Promise<TransactionResponse> {
   try {
-    const gas = await getGasPrice(contract.provider);
+    const gas = await getGasPrice(
+      contract.provider,
+      process.env.PRIORITY_FEE_SCALER && Number(process.env.PRIORITY_FEE_SCALER),
+      process.env.MAX_FEE_PER_GAS_SCALER && Number(process.env.MAX_FEE_PER_GAS_SCALER)
+    );
```

Comment:
Thoughts on something like this where we have a chain-specific one and then an omnichain one to fall back to? That way we can tune it for each chain if we like. Something like:

```suggestion
    const priorityFeeScaler = Number(process.env[`PRIORITY_FEE_SCALER_${chainId}`] || process.env.PRIORITY_FEE_SCALER) || undefined;
    const maxFeePerGasScaler = Number(process.env[`MAX_FEE_PER_GAS_SCALER_${chainId}`] || process.env.MAX_FEE_PER_GAS_SCALER) || undefined;

    const gas = await getGasPrice(
      contract.provider,
      priorityFeeScaler,
      maxFeePerGasScaler
    );
```

Reply:
Yeah it'd be useful for this to be per-chain configurable.

Reply:
Great idea!